ui/src/app/pages/analysis/interactive-notebook.spec.tsx

```diff
@@ -49,6 +52,8 @@ beforeEach(async () => {
   jest.mock('app/services/swagger-fetch-clients');
   mockAppsApi = jest.spyOn(swaggerClients, 'appsApi');
   mockNotebooksApi = jest.spyOn(swaggerClients, 'notebooksApi');
+  helpSidebarConfigIdForUIApp[UIAppType.RSTUDIO] = rstudioConfigIconId;
```

Comment:
hmm.  We shouldn't need to do this.  There may be something wrong about how we're using the record here.  I'll think about this a bit.

Reply:
I made https://github.com/all-of-us/workbench/pull/8232 to help look into this ... certainly low priority